src/Mod/Material/importFCMat.py

```diff
@@ -97,9 +97,9 @@ def decode(name):
 # as we had and we might will have problems again and again
 # https://github.com/berndhahnebach/FreeCAD_bhb/commits/materialdev
 
-def read(filename):
+def read(filename, library):
```

Comment:
Unfortunately this code will only read materials from the standard library, not user materials, etc. Specifying the library is not required when the full path is required, and when using the old system is not recommended.

This change will also break the many add on workbenches that call this function.

Reply:
Thanks for the review. Reverted changes, now calling this function from ArchMaterial.py with absolute material path.